### ISSUE TITLE
Feature request: setting to allow looping of animated gifs

### ISSUE THREAD

**Issue by pmonks** [Dimillian/IceCubesApp#1051]
Some animated gifs, such as [this one](https://sfba.social/@pmonks/109921792706083607), are very short, and only function as a communication device if they are played continuously on loop.

That said, automatically looping gifs may be annoying for some users, so I would like to propose a new configuration setting that controls whether animated gifs are looped or not.

Additionally, it may be worth considering whether there should be a (configurable) cutoff for this looping behaviour i.e. animated gifs longer than X seconds in duration are not looped, regardless of whether auto-looping is enabled or not.

Finally, if links to animated gifs hosted elsewhere (i.e. not on Mastodon) are displayed inline by the app, they should also respect these new configuration settings.

**Comment by stefan-scholl** [comment]
Yes, this being currently different from the official web interface of Mastodon (and maybe other social media clients) bothers me.

As the example link of the first comment isn't available anymore, I provide this short video: https://mastodon.social/@Stefan_S_from_H/110779659267804720

Mastodon web marks it with the label “GIF” and plays it in a loop on mouse over: 
![image](https://github.com/user-attachments/assets/a0e9a46d-7009-47dd-9680-e4cb57af7414)
